README.md

```diff
@@ -7,42 +7,52 @@
 
 ## Overview
 
-This add-on integrates Tsh into your [DDEV](https://ddev.com/) project.
+This add-on integrates [Teleport](https://goteleport.com/) and its command line tool, [tsh](https://goteleport.com/docs/connect-your-client/tsh/) into your [DDEV](https://ddev.com/) project.
```

Comment:
Thanks for the link. Just a little more context on what teleport is for the naive ones among us, just a sentence, might help people evaluate whether this was relevant to them.  But I imagine they would only be looking if they already knew it was relevant...